packages/amplify-category-api/provider-utils/__tests__/apigw-walkthroughs.test.js

```diff
@@ -3,11 +3,13 @@ const rewire = require('rewire');
 const walkthrough = rewire('../awscloudformation/service-walkthroughs/apigw-walkthrough.js');
 
 const validatePathName = walkthrough.__get__('validatePathName');
-const stubOtherPaths = [{ name: '/other/path' }, { name: '/sub/path' }];
+const checkForPathOverlap = walkthrough.__get__('checkForPathOverlap');
+const stubOtherPaths = [{ name: '/other/path' }, { name: '/sub/path' }, { name: '/path/{with}/{params}' }];
 
 test('validatePathName_validPath', () => {
   expect(validatePathName('/some/path', stubOtherPaths)).toBeTruthy();
   expect(validatePathName('/path/{with}/{params}', stubOtherPaths)).toBeTruthy();
+  expect(validatePathName('/', stubOtherPaths)).toBeTruthy();
```

Comment:
should be `.toBe(true);`

validatePathName always returns a truthy value.

Reply:
👍